tests/test_trajectory.py

```diff
@@ -203,14 +203,16 @@ def test_orientations(self):
             with self.assertRaises(ValueError):
                 # This should fail since it's using 2d positions
                 traj[0].orientations = [[0, 0], [0, 0]]
+        except AttributeError:
+            pass
 
```

Comment:
For now, I added these `try` blocks on several tests since the `TrajectoryTest` class is subclassed on various specific format test modules. This prevents some of those other tests to fail. However, in the future, it might me better to refactor these tests to avoid the `try` blocks.